generate-pages-from-markdown.js

```diff
@@ -22,7 +22,19 @@ var options = {
 var route_array = []
 dive(mark_dir, route_array);
 console.log("Routing:\n" + toSource(route_array));
-
+var contributors_route = { id:"contributing",
+    alias:"Contributors",
+    type:"route",
+    route:"contributors",
+    items:[ { id:"contributors",
+        alias:"Contributors",
+        type:"route",
+        route:"contributors.contributors" },
```

Comment:
Can we make this a flat route instead of nested?  e.g. contributors is a top level route instead of contributors.contributors?